src/components/Header.vue

```diff
@@ -1,40 +1,68 @@
 <script lang="ts">
     import { defineComponent } from 'vue';
+    import CartFlyout from './CartFlyout.vue';
+    import { CartItemInterface } from '../types';
 
     export default defineComponent({
-    name: 'Header',
-    props: {
-        mobileMenuOpen: Boolean,
-        cartOpen: Boolean,
-        totalQuantity: {
-            type: Number,
-            default: 0,
-        },
-    },
+      components: {CartFlyout},
+      name: 'Header',
+      props: {
+          mobileMenuOpen: Boolean,
+          cartOpen: Boolean,
+          totalQuantity: {
+              type: Number,
+              default: 0,
+          },
+          cartItems: {
+            type: Array as () => CartItemInterface[],
+            required: true,
+          },
+      },
+      setup(props, context) {
+        const handleRemoveFromCart = (itemId: number) => {
+          console.log('remove from cart', itemId);
+          context.emit('remove-from-cart', itemId);
+        };
+
+        return {
+          handleRemoveFromCart,
+        };
+      },
     });
 </script>
 
 <template>
-    <header class="py-5 px-6 flex justify-between items-center">
-      <div class="flex justify-between items-center gap-4">
-        <div class="md:hidden">
-          <button class="flex justify-center items-center" @click="$emit('toggle-mobile-menu')">
-            <img src="../assets/icon-menu.svg" alt="menu" />
-          </button>
-        </div>
-        <img class="self-center pb-1" src="../assets/logo.svg" alt="sneakers" />
+    <header class="py-5 md:py-0 px-6 md:px-0 flex justify-between items-center border-b-neutral-grey border-b-2 relative">
+      <div class="flex items-center">
+        <button class="md:hidden" @click="$emit('toggle-mobile-menu')">
+          <img src="../assets/icon-menu.svg" alt="menu" />
+        </button>
+        <img class="self-center pb-1 ml-4 md:ml-0" src="../assets/logo.svg" alt="sneakers" />
```

Comment:
Relative paths can be error-prone when the codebase grows and the file structure changes. It would be better to use absolute paths or module paths. It could be `@/assets/logo.svg`, assuming you have set up the @ alias in your TypeScript configuration file.

Reply:
It's ok for now with src then, for the future:
https://vuejs.org/guide/typescript/overview.html#configuring-tsconfig-json
https://towardsthecloud.com/set-absolute-path-typescript